authentication/auth.go

```diff
@@ -300,14 +300,14 @@ func generateTokenFromClaims(cl EqClaims) (string, *TokenError) {
 		return "", &TokenError{Desc: "Error signing and encrypting JWT", From: err}
 	}
 
-	fmt.Printf("Created signed/encrypted JWT: %v", token)
+	log.Println("Created signed/encrypted JWT:", token)
 
 	return token, nil
 }
 
 // GenerateTokenFromDefaults coverts a set of DEFAULT values into a JWT
 func GenerateTokenFromDefaults(url string) (string, *TokenError) {
-	log.Println("Getting ...", url)
+	log.Println("Getting url parameter...", url)
```

Comment:
Im still not sure it makes sense.
It should also be just before the action to load the schema